core/src/test/scala/integration/kafka/admin/ConfigCommandIntegrationTest.scala

```diff
@@ -43,8 +45,9 @@ class ConfigCommandIntegrationTest extends QuorumTestHarness with Logging {
       "--add-config", "security.inter.broker.protocol=PLAINTEXT"))
   }
 
-  @Test
-  def shouldExitWithNonZeroStatusOnZkCommandAlterUserQuota(): Unit = {
+  @ParameterizedTest
+  @ValueSource(strings = Array("zk"))
+  def shouldExitWithNonZeroStatusOnZkCommandAlterUserQuota(quorum: String): Unit = {
```

Comment:
Can you confirm that you need a ZK cluster to perform this test? If not can we move it to the suite that doesn't inherent from `QuorumTestHarness`?